SPDX-License-Identifier: BSD-3-Clause

Use ram's Cabal version macro after building cryptostore against ram.

Index: src/Crypto/Store/Util.hs
--- src/Crypto/Store/Util.hs.orig
+++ src/Crypto/Store/Util.hs
@@ -39,7 +39,7 @@
 
 -- | Reverse a bytearray.
 reverseBytes :: ByteArray ba => ba -> ba
-#if MIN_VERSION_memory(0,14,18)
+#if MIN_VERSION_ram(0,14,18)
 reverseBytes = B.reverse
 #else
 reverseBytes = B.pack . reverse . B.unpack
